src/input.cpp

```diff
@@ -1896,26 +1896,19 @@ const hotkey_queue &hotkey_queue::alphabets()
     return *queue;
 }
 
-const hotkey_queue &hotkey_queue::alpha_digits()
+const hotkey_queue &hotkey_queue::create_from_available_hotkeys( input_context &ctxt )
```

Comment:
This will always return the queue created for the first input context this function is called with, because `queue` is static.